test/core.controller.tests.js

```diff
@@ -480,23 +471,27 @@ describe('Chart.Controller', function() {
 	});
 
 	describe('controller.destroy', function() {
-		it('should remove the resizer element when responsive: true', function() {
+		it('should remove the resizer element when responsive: true', function(done) {
 			var chart = acquireChart({
 				options: {
 					responsive: true
 				}
 			});
 
-			var wrapper = chart.chart.canvas.parentNode;
-			var resizer = wrapper.firstChild;
+			setTimeout(function() {
```

Comment:
I've looked around the element-resize-detector code and it seems like it defers attaching the resize element. I'm not sure exactly why it's implemented like that.